dm/config/task.go

```diff
@@ -326,6 +329,10 @@ func (m *LoaderConfig) adjust() error {
 		m.PoolSize = defaultPoolSize
 	}
 
+	if m.Dir != "" && m.SortingDirPhysical == "" {
+		m.SortingDirPhysical = m.Dir
```

Comment:
Why put the `sorting-dir` with `source-dir`? Lightning will prefer they are in different disk. Maybe we can set another default `sorting-dir`?

Reply:
I don't know how to do it in code 😂 and user may be unexpected to see another disk is occupied

Reply:
👌 Maybe just leave some comments to demonstrate the risk here?